<?xml version="1.0" ?>
<gazebo version="1.2">
  <world name="default">

    <!-- A ground plane -->
    <include>
      <uri>model://ground_plane</uri>
    </include>

    <!-- A global light source -->
    <include>
      <uri>model://sun</uri>
    </include>

    <!-- a static object -->
    <model name="cylinder_model">
      <static>true</static>
      <link name="cylinder_link">
        <visual name="cylinder_visual">
          <pose>0 0 0.5 0 0 0</pose>
          <geometry>
            <cylinder>
            <radius>0.5</radius>
            <length>1.0</length>
            </cylinder>
          </geometry>
        </visual>
        <collision name="cylinder_collision">
          <pose>0 0 0.5 0 0 0</pose>
          <geometry>
            <cylinder>
            <radius>0.5</radius>
            <length>1.0</length>
            </cylinder>
          </geometry>
        </collision>
      </link>
    </model>

    <!-- a ray sensor -->
    <include filename="hokuyo.model" model_pose="-1.0 0 0.5 0 0 0"/>
  </world>
</gazebo>
